libc/spec/stdc.td

```diff
@@ -967,6 +967,13 @@ def StdC : StandardSpec<"stdc"> {
               RetValSpec<IntType>,
               [ArgSpec<IntType>, ArgSpec<FILEPtr>]
           >,
+          FunctionSpec<
+              "vasprintf",
+              RetValSpec<IntType>,
+              [ArgSpec<CharRestrictedPtrPtr>,
```

Comment:
Could you point me to the spec that provides the restrict attribute? Although I think it is fine to have this flag, the manpage I checked does not seem to have it.

Reply:
My man pages state:
```
       int asprintf(char **restrict strp, const char *restrict fmt, ...);
       int vasprintf(char **restrict strp, const char *restrict fmt,
                     va_list ap);
```